### PR TITLE
#226 add common api for structure creating

### PR DESCRIPTION
**Pull request by TomasLight**

### CODE REVIEW

File: packages/ketcher-react/src/script/builders/ketcher/ButtonsConfig.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { ButtonConfig } from './ButtonConfig'
+import { ButtonName } from './ButtonName'
+
+export type ButtonsConfig = {
+  [buttonName in ButtonName]: ButtonConfig
+}
```
Comment (TomasLight) [inline packages/ketcher-react/src/script/builders/ketcher/ButtonsConfig.ts:6]: will be corerected in different PR

File: packages/ketcher-react/src/script/ui/state/modal/form.js [hunk 2]
```diff
@@ -91,7 +90,7 @@ export const formsState = {
     valid: true,
     result: {
       filename: 'ketcher',
-      format: SupportedFormat.Mol
+      format: 'mol'
```
Comment (AndreiMazol) [inline packages/ketcher-react/src/script/ui/state/modal/form.js:93]: Why do you use magic string instead of enum type?

File: packages/ketcher-react/src/script/ui/dialog/mainmenu/save.jsx [hunk 3]
```diff
@@ -176,7 +186,7 @@ class Save extends Component {
             schema={this.saveSchema}
             init={{
               filename,
-              format: this.isRxn ? SupportedFormat.Rxn : SupportedFormat.Mol
+              format: this.isRxn ? 'rxn' : 'mol'
```
Comment (AndreiMazol) [inline packages/ketcher-react/src/script/ui/dialog/mainmenu/save.jsx:189]: Could you remove magic strings?

**Review by AndreiMazol — APPROVED** [review]
